lib/ffaker/phone_number_sn.rb

```diff
@@ -64,15 +64,9 @@ def homework_number
     # Generates a random phone number mobile or home or work
     #
     def phone_number
-      case rand(0..9)
-      when 0 then homework_number
-      when 9 then homework_number
-      when 3 then mobile_number
-      when 5 then mobile_number
-      when 6 then mobile_number
-      when 8 then mobile_number
-      else        homework_number
-      end
+      return homework_number if rand(0..1).zero?
```

Comment:
is not about this pr but, ArrayUtils.rand is deprecated.
```suggestion
      return homework_number if fetch_sample(0..1).zero?
```